dbt/adapters/athena/lakeformation.py

```diff
@@ -20,10 +22,42 @@
 logger = AdapterLogger("AthenaLakeFormation")
 
 
+def _column_tags_to_remove(
+    lf_tags_columns: List[ColumnLFTagTypeDef], lf_inherited_tags: Set[str]
+) -> Dict[str, Dict[str, List[str]]]:
+    to_remove = {}
+
+    for column in lf_tags_columns:
+        non_inherited_tags = [tag for tag in column["LFTags"] if not tag["TagKey"] in lf_inherited_tags]
+        for tag in non_inherited_tags:
+            tag_key = tag["TagKey"]
+            tag_value = tag["TagValues"][0]
+            if tag_key not in to_remove:
+                to_remove[tag_key] = {tag_value: [column["Name"]]}
+            elif tag_value not in to_remove[tag_key]:
+                to_remove[tag_key][tag_value] = [column["Name"]]
+            else:
+                to_remove[tag_key][tag_value].append(column["Name"])
+
+    return to_remove
+
+
+def _table_tags_to_remove(
+    lf_tags_table: list[LFTagPairTypeDef], lf_tags: Optional[Dict[str, str]], lf_inherited_tags: set[str]
+):
```

Comment:
let's add type annotation here as well

Reply:
Ok added. Fixed the type checking errors as well.